### PR TITLE
add the jenkins build to source control

### PR DESCRIPTION
**Pull request by wwwtyro**
DO NOT MERGE

### CODE REVIEW

**Comment by wplu** [comment]
perhaps rename script to jenkins-build.sh or build-jenkins.sh

**Comment by wplu** [comment]
perhaps set -e

**Comment by wwwtyro** [comment]
Excellent points Phil, thank you. Will do.

**Comment by wwwtyro** [comment]
test this

**Comment by wwwtyro** [comment]
test this

**Comment by wwwtyro** [comment]
test this

**Comment by wwwtyro** [comment]
test this

**Comment by wwwtyro** [comment]
test this

**Comment by wwwtyro** [comment]
test this

File: jenkins-build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+unset EDITOR # so we don't fail a cli test :\
+gvm use go1.4.2
+go version
+docker version
+export GOPATH=$WORKSPACE/gopath
+export PATH=$PATH:$GOPATH/bin
```
Comment (wplu) [inline jenkins-build.sh]: reverse this

File: jenkins-build.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+sudo umount /exports/serviced_var || true
```
Comment (wplu) [inline jenkins-build.sh]: umount /exports/serviced_var_volumes

File: jenkins-build.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+sudo rm /tmp/serviced-root/var/isvcs/* -Rf
+sudo rm /tmp/serviced-test -Rf
+docker ps -a -q | xargs --no-run-if-empty docker rm -f
```
Comment (wplu) [inline jenkins-build.sh]: rm -fv

**Comment by wplu** [comment]
retest this please

**Pull request merged**